pkgs/development/python-modules/pgzero/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3Packages
+}:
+
+buildPythonPackage rec {
+  pname = "pgzero";
+  version = "1.2.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-jK3AIPAoy6w+DL07uTEaHARfHe7ax5F/9DP5hsOOYQY=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    enum34
+    numpy
+    pygame
+  ];
+
+  pythonImportsCheck = [ "pgzero" ];
+
+  meta = with lib; {
+    description = "A zero-boilerplate 2D games framework";
+    homepage = "https://pypi.org/project/pgzero/";
```

```suggestion
    homepage = "https://github.com/lordmauve/pgzero";
```
The source code page is usually way more helpful than pypi